src/ray/raylet/scheduling_queue.cc

```diff
@@ -6,49 +6,90 @@ namespace ray {
 
 namespace raylet {
 
+bool SchedulingQueue::TaskQueue::Append(const TaskID &task_id, const Task &task) {
+  if (map_.find(task_id) != map_.end()) {
+    // This shouldn't happen.
+    return false;
+  }
+
+  auto list_iter = list_.insert(list_.end(), task);
+  map_[task_id] = list_iter;
+  return true;
+}
+
+bool SchedulingQueue::TaskQueue::Remove(const TaskID &task_id) {
+  auto iter = map_.find(task_id);
+  if (iter == map_.end()) {
+    // not exist.
```

Comment:
We try to make all comments be complete, capitalized sentences. E.g., "Specified task was not found in the map". In this particular case this is also obvious from the code, so this comment is unnecessary.

Reply:
sure.